.repo/sample-list-generator/src/utils/manifest.ts

```diff
@@ -1,11 +1,32 @@
 import fs from 'fs/promises';
-import { ManifestData } from '../types';
+import { dirname } from 'path';
+import { ManifestData, LocaleData } from '../types';
+const localeRegex = /__MSG_([^_]*)__/
 
 export const getManifest = async (
   manifestPath: string
 ): Promise<ManifestData> => {
   const manifest = await fs.readFile(manifestPath, 'utf8');
   const parsedManifest = JSON.parse(manifest);
 
+  if (manifest.includes('__MSG_')) {
```

Comment:
It makes me really uncomfortable searching through an unparsed file like this 😅  - would you be open to removing the if here, and just move loading of the messages.json file to when we find a value that starts with MSG? It does mean we might load the file multiple times unless you want to do some more tweaking but I don't think that's the end of the world.

Reply:
abstracted in bd6bf32b0b4b1b101899937aae785d654efcf36b.

For the sake of localization on dcc,  we could potentially localize the metadata for the samples extension and update this to take a langauge flag as an argument to load that language, rather than the currently hardcoded english.